src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,179 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import tasks.Task;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Todo;
+
+import dukeException.DukeException;
+import dukeException.DukeIOBException;
+
+public class Duke {
+
+    public Duke() {
+
+    }
```

Comment:
Do consider removing empty methods, if not you could refactor some of your code into this. :)